.ci_support/osx_64_numpy1.18python3.7.____cpython.yaml

```diff
@@ -1,13 +1,13 @@
 MACOSX_DEPLOYMENT_TARGET:
-- '10.14'
+- '10.9'
```

Comment:
@xhochy I think this leads to failures in the MacOS builds. Not sure the best way to fix it.

Reply:
This needs to go into `recipe/conda_build_config.yaml`. All the `.ci_support` files are auto-generated.

Reply:
Shouldn't 10.12 be sufficent?

Reply:
Added to recipe/conda_build_config.yaml.

I don't remember the details but when I updated to onnxruntime 1.9.0, I think MacOS 10.14 was the lowest version that worked. I think lower versions had some missing C++ standard library support.